.editorconfig

```diff
@@ -0,0 +1,29 @@
+# EditorConfig is awesome: https://EditorConfig.org
```

Comment:
I disagree. .editorconfig is intended to ensure any .editorconfig compliant editor (most modern editors/IDEs) comply to the standards used in the repo. otherwise you will spurious indent changes but contributors.

Reply:
Synthea uses checkstyle for this purpose.